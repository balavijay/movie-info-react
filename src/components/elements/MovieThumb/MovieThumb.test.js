import React from 'react';
import { shallow, mount } from 'enzyme';


import MovieThumb from './MovieThumb';

describe ("MovieThumb Component " , () => {
   const  app = shallow(<MovieThumb />);
    test("MovieThumb render" , () => {
        expect(app).toMatchSnapshot();
    })

    

 
});