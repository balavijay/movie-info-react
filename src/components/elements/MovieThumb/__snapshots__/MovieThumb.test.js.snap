// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieThumb Component  MovieThumb render 1`] = `ShallowWrapper {}`;
